src/Codeception/Test/Cest.php

```diff
@@ -114,7 +117,7 @@ protected function executeHook($I, string $hook): void
         }
     }
 
-    protected function executeBeforeMethods($testMethod, $I): void
+    protected function executeBeforeMethods(?string $testMethod, $I): void
```

Comment:
Why is it nullable? It doesn't make sense.

Reply:
`$testMethod` is nullable in PHPUnit 9.5 and lower https://github.com/sebastianbergmann/phpunit/blob/9550b9cb7461584ed4beeb74d119d01808000a8c/src/Util/Test.php#L328 .

do I prevent it from being nullable here?

Reply:
That means, that PHPUnit method can be used for something else. We don't use it for that purpose.

Cest methods must have names, so they are not nullable,.

Reply:
thanks. Done.